com.valvesoftware.Steam.json

```diff
@@ -71,6 +71,63 @@
 		"sha256": "c1cfd4e1d4d708c031d60801e527abc9b6d34b85f2ffa2cadd21f75ff38151cd"
 	    }]
 	},
+        {
+            "name": "systemd",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Drootprefix=/app",
+
+                "-Denvironment-d=false",
+                "-Dbinfmt=false",
+                "-Dcoredump=false",
+                "-Dlogind=false",
+                "-Dhostnamed=false",
+                "-Dlocaled=false",
+                "-Dmachined=false",
+                "-Dportabled=false",
+                "-Dnetworkd=false",
+                "-Dtimedated=false",
+                "-Dtimesyncd=false",
+                "-Dremote=false",
+                "-Dfirstboot=false",
+                "-Drandomseed=false",
+                "-Dbacklight=false",
+                "-Dvconsole=false",
+                "-Dquotacheck=false",
+                "-Dsysusers=false",
+                "-Dtmpfiles=false",
+                "-Dimportd=false",
+                "-Dhwdb=false",
+                "-Drfkill=false",
+                "-Dman=false",
+                "-Dhtml=false",
+
+                "-Dpam=false",
+                "-Dmicrohttpd=false",
+
+                "-Dbashcompletiondir=no",
+                "-Dzshcompletiondir=no"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/systemd/systemd/archive/v239.tar.gz",
+                    "sha256": "8a11b1b07d620f4c06a16e95bba4dd2a97e90efdf2a5ba47ed0a935085787a14"
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/lib/systemd",
+                "/lib/sysctl.d",
+                "/lib/udev/rules.d",
+                "/lib/modprobe.d",
+                "/lib/kernel",
+                "/share/polkit-1",
+                "/include",
+                "/lib/pkgconfig",
```

Comment:
pkgconfig is used for detection on whether libsystemd is installed so this will break gamemode